src/main/kotlin/com/exactpro/th2/ws/client/api/impl/WebSocketClient.kt

```diff
@@ -237,10 +238,30 @@ class WebSocketClient(
         return null
     }
 
-    private class WebSocketClientSettings(private val builder: WebSocket.Builder) : IClientSettings {
+    private class WebSocketClientSettings(private val builder: WebSocket.Builder, private val uriBuilder: URIBuilder) : IClientSettings {
         override fun addHeader(name: String, value: String) {
             builder.header(name, value)
         }
+
+        override fun addQueryParam(name: String, value: String) {
+            uriBuilder.addQueryParam(name, value)
+        }
+    }
+
+    private class URIBuilder(private val baseUri: String) {
+        private val queryBuilder: StringBuilder = StringBuilder("")
+
+        fun addQueryParam(name: String, value: String) {
+            queryBuilder.append("$name=$value&")
```

Comment:
`name` and `value` should be encoded via `URLEncoder`, otherwise creating a URI from this string will fail if it contains control characters

Reply:
Done